include/bluetooth/services/ras.h

```diff
@@ -329,15 +337,15 @@ typedef void (*bt_ras_rreq_rd_ready_cb_t)(struct bt_conn *conn, uint16_t ranging
  */
 typedef void (*bt_ras_rreq_rd_overwritten_cb_t)(struct bt_conn *conn, uint16_t ranging_counter);
 
-/** @brief Ranging data get complete callback. Called when ranging data get procedure has completed.
+/** @brief Ranging data complete callback. Called when complete ranging data has been received from
```

Comment:
Brief is usually just one sentence in imperative format.

Reply:
Can you give a suggestion on how to rewrite it?

Reply:
Oh, this is a hard one. Receive complete ranging data? Then an additional line below to describe more?

Reply:
Shall we leave it as-is then? I've only copied the format of the other callbacks in this file. See for instance the docs for `bt_ras_rreq_rd_overwritten_cb_t` and `bt_ras_rreq_rd_ready_cb_t`

Reply:
Fair enough.